### ISSUE TITLE
Variable substitution bug

### ISSUE THREAD

**Issue by dremekie** [AnWeber/httpyac#99]
```
POST https://httpbin.org/anything HTTP/1.1
{
  "template": "My \{\{someVerb\}\} template!!"
}
```

results in:

```
ERROR: ReplaceVariableHook: javascript failed
ERROR: BeforeRequestHook: requestVariableReplacer failed
ERROR: https://httpbin.org/anything
{
  "url": "https://httpbin.org/anything",
  "method": "POST",
  "http2": false,
  "headers": {},
  "body": "{\n  \"template\": \"My \\{\\{someVerb\\}\\} template!!\"\n}",
  "cookieJar": {
    "version": "tough-cookie@4.0.0",
    "storeType": "D_",
    "rejectPublicSuffixes": true,
    "cookies": []
  }
}
ReferenceError - someVerb is not defined
ReferenceError: someVerb is not defined
	at Object.<anonymous> (/home/dwayne/github/mesh-server/test/http/intel/test.http:7:20)
	at TE (/home/dwayne/.vscode-server/extensions/anweber.vscode-httpyac-4.5.0/dist/extension.js:105:91651)
	at Hc (/home/dwayne/.vscode-server/extensions/anweber.vscode-httpyac-4.5.0/dist/extension.js:105:92017)
	at Object.Rde [as action] (/home/dwayne/.vscode-server/extensions/anweber.vscode-httpyac-4.5.0/dist/extension.js:106:43256)
	at yle.trigger (/home/dwayne/.vscode-server/extensions/anweber.vscode-httpyac-4.5.0/dist/extension.js:105:76730)
	at async wu (/home/dwayne/.vscode-server/extensions/anweber.vscode-httpyac-4.5.0/dist/extension.js:105:100312)
	at async wgt (/home/dwayne/.vscode-server/extensions/anweber.vscode-httpyac-4.5.0/dist/extension.js:105:128074)
	at async Object.Mpe [as action] (/home/dwayne/.vscode-server/extensions/anweber.vscode-httpyac-4.5.0/dist/extension.js:105:127972)
	at async wE.trigger (/home/dwayne/.vscode-server/extensions/anweber.vscode-httpyac-4.5.0/dist/extension.js:105:76713)
	at async Xd (/home/dwayne/.vscode-server/extensions/anweber.vscode-httpyac-4.5.0/dist/extension.js:105:88250)
ERROR: ExecuteHook: httpClient failed
ERROR: ReferenceError - someVerb is not defined
ReferenceError: someVerb is not defined
	at Object.<anonymous> (/home/dwayne/github/mesh-server/test/http/intel/test.http:7:20)
	at TE (/home/dwayne/.vscode-server/extensions/anweber.vscode-httpyac-4.5.0/dist/extension.js:105:91651)
	at Hc (/home/dwayne/.vscode-server/extensions/anweber.vscode-httpyac-4.5.0/dist/extension.js:105:92017)
	at Object.Rde [as action] (/home/dwayne/.vscode-server/extensions/anweber.vscode-httpyac-4.5.0/dist/extension.js:106:43256)
	at yle.trigger (/home/dwayne/.vscode-server/extensions/anweber.vscode-httpyac-4.5.0/dist/extension.js:105:76730)
	at async wu (/home/dwayne/.vscode-server/extensions/anweber.vscode-httpyac-4.5.0/dist/extension.js:105:100312)
	at async wgt (/home/dwayne/.vscode-server/extensions/anweber.vscode-httpyac-4.5.0/dist/extension.js:105:128074)
	at async Object.Mpe [as action] (/home/dwayne/.vscode-server/extensions/anweber.vscode-httpyac-4.5.0/dist/extension.js:105:127972)
	at async wE.trigger (/home/dwayne/.vscode-server/extensions/anweber.vscode-httpyac-4.5.0/dist/extension.js:105:76713)
	at async Xd (/home/dwayne/.vscode-server/extensions/anweber.vscode-httpyac-4.5.0/dist/extension.js:105:88250)
```

**Comment by AnWeber** [comment]
yes, of course. I had forgotten about it. When I thought about the change for httpyac/httpyac.github.io#7, I noticed the spot, but I forgot about it when I was programming. I'm sorry.

**Comment by AnWeber** [comment]
A workaround would be possible in the following form

```
{{@request
request.body = `
{
  "template": "My {{someVerb}} template!!"
}`;
}}

POST https://httpbin.org/anything

```

**Comment by dremekie** [comment]
I'll try the workaround. In future, will you be making the change to handle `"My \{\{someVerb\}\} template!!"` ?

**Comment by AnWeber** [comment]
It was an oversight. I just need to change the timing of the call. But have overlooked it.

**Comment by AnWeber** [comment]
I have now fixed the bug with v4.6.0. I had unfortunately not considered the escaping when replacing the variables deep.
